src/components/FeedbackConstants.js

```diff
@@ -0,0 +1,29 @@
+export const FEEDBACK_CODES = Object.freeze({
+  1: "Article Reader",
```

Comment:
Does this have to be manually kept in sync with the FeedbackComponent? Then we should make a BIG BOLD note about it here. I remember we had to populate the DB with these values independently so this is why I wonder.